src/stories/components/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, useMemo } from 'react';
+
+import * as S from './styles';
+import { BreadcrumbsProps } from './types';
+
+export const Breadcrumbs: FC<BreadcrumbsProps> = ({ allLinks = false, items }) => {
+  const breadcrumbs = useMemo(() => {
+    if (items.length > 0) {
+      return items?.map((item, index) => {
+        if (index + 1 === items.length && !allLinks) {
+          return <S.LastText key={item.text}>{item.text}</S.LastText>;
+        }
+
+        return (
+          <S.LinkLabel key={item.text} href={item.redirectLink}>
+            {item.text}
+          </S.LinkLabel>
+        );
+      });
+    }
+
+    return [];
+  }, [items, allLinks]);
+
+  return (
+    <S.Breadcrumbs separator="›" aria-label="breadcrumb">
```

Comment:
Sugestão: seria interessante passar o `separator` como prop opcional e não obrigatória do componente com o valor default `"›"` pois que for utilizar por passar outro separador como uma barra por exemplo `"/"`

Reply:
Ajustado!!